### ISSUE TITLE
Add tests

### ISSUE THREAD

**Issue by mcmunder** [mcmunder/eaternity-react#7]
... this one will not be closed for a while 😝

**Comment by mklarmann** [comment]
https://news.ycombinator.com/item?id=13128146

**Comment by mcmunder** [comment]
yep, I am planning to spend quite a few hours/days learning jest + enzyme or tape + enzyme...

**Comment by mklarmann** [comment]
Enzyme haha - das ist was für den Biologen!
